hera_cal/redcal.py

```diff
@@ -218,6 +224,28 @@ def split_bls(bls):
         lengths = [np.mean([np.linalg.norm(antpos[bl[1]] - antpos[bl[0]]) for bl in gp]) for gp in reds]
         reds = [gp for gp, l in zip(reds, lengths) if ((min_bl_cut is None or l > min_bl_cut)
                                                        and (max_bl_cut is None or l < max_bl_cut))]
+    
+    if max_dims is not None:
+        while True:
+            # Compute idealized antenna positions from redundancies
```

Comment:
I think it would be helpful to put a reference here to where idealized antenna positions are described and maybe mention here that they are antenna positions with some arbitrary linear transformation (that's what I think they are based on reading other comments/code but if not say something else ;) ).